base/show.jl

```diff
@@ -1536,7 +1541,11 @@ function isidentifier(s::AbstractString)
     is_id_start_char(c) || return false
     return all(is_id_char, rest)
 end
-isidentifier(s::Symbol) = isidentifier(string(s))
+
+function isidentifier(s::Symbol)
+    str = string(s)
+    return !isidentifier(str) ? false : str == Unicode.normalize(str; stable=true, compose=true)
```

Comment:
This is not correct, because it doesn't do Julia's character normalization.  That's why I suggested:
```suggestion
    ct(codepoint::UInt32) = get(Unicode._julia_charmap, codepoint, codepoint)
    isidentifier(str) && str == Unicode.normalize(str; stable=true, compose=true, chartransform=ct)
```
(Note that `&&` is short-circuiting in Julia, so the `normalize` function is not called if `isidentifier` returns `false`.  So there is no need for `... ? ... : ...`.)

Reply:
this is resulting in segmentation fault.I am trying to resolve it.

Reply:
Try making the `ct` function in global scope so that it is a constant:
```jl
_julia_chartransform(codepoint::UInt32) = get(Unicode._julia_charmap, codepoint, codepoint)
function isidentifier(s::Symbol)
    str = string(s)
    isidentifier(str) && str == Unicode.normalize(str; stable=true, compose=true, chartransform=_julia_chartransform)
end
```

Reply:
its still giving segmentation fault even if declared globally

Reply:
Yeah, I figured that wouldn't help — using a local function works at the REPL.

It must be some kind of bootstrapping thing, where `isidentifier` is being called early in to boot process at a point where something else (`get`? `Unicode`?) hasn't been defined yet.

Reply:
@staticfloat, do you know why this might crash during bootstrapping but not if you enter the same function at the REPL?

As far as I can tell, `dict.jl` and `unicode.jl` are loaded before `show.jl`, so everything needed here should be defined.

Reply:
I don't have any idea off the top of my head, but if you can catch it inside of `gdb` and print out the disassembly we might get a better idea.

Reply:
@stevengj I posted the crash log 😅, seems like @11happy is also working on the same issue